### DISCUSSION TITLE
Can I using Emotion with GSAP

### DISCUSSION THREAD

**Discussion by rahXephonz** [emotion-js/emotion#2628] [Q&A] [answered]
Hello everyone👋

Just asking some question, i have an idea to combine Emotion with GSAP. Can someone give me link about some reference please?

**Comment by srmagura** [comment, marked as answer]
Here is GSAP's documentation on usage with React (assuming you are using React): https://greensock.com/react

You should be able to use Emotion and GSAP together without the two libraries knowing about each other. 

Is there a specific problem you are trying to solve?

**Reply by rahXephonz** [reply]
Nope sir, but in my Case when i trying gsap with typescript some files or fn are deprecated
